docs/UserGuide.md

```diff
@@ -240,53 +240,53 @@ or decremented respectively by the value of your input.
 This command locates guests whose details contain any of the given keywords. This allows you to search for 
 guests with any of the specific guests' field.
 
-This command is typically used when searching for a guest of a specific characteristic. This command also allows for
+This command is typically used when searching for a guest of a **specific characteristic**. This command also allows for
 filtering of the guest list displayed. For example, the command `find no` will display all 
-the rooms that have room clean statuses set to "no".
+the rooms that have room clean statuses set to _"no"_.
 
 #### Format: `find KEYWORD [MORE_KEYWORDS]`
 
 #### Constraints of parameters:
-* The search is case-insensitive. e.g `hans` will match `Hans`
+* The search is **case-insensitive**. e.g `hans` will match `Hans`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Guests matching at least one keyword will be returned (i.e. `OR` search).
+* **Only full words** will be matched e.g. `Han` will not match `Hans`
+* Guests matching **at least one** keyword will be returned (i.e. `OR` search).
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 
 #### Examples:
 * `find John` returns `john` and `John Doe`
-* `find alex 99272758 03-68` returns `Alex Yeoh`, `Bernice Yu`, `Charlotte Oliveiro` (refer to the image below for the details of the guests)<br>
+* `find alex 99272758 03-68` returns `Alex Yeoh`, `Bernice Yu`, `Charlotte Oliveiro` (refer to the image below for the details of the _guests_)<br>
 <br>
 
   ![result for 'find alex 99272758 03-68'](images/findAlex9927275803-68.png)
 
 #### Common mistakes:
-* Searching for guests using partial field characteristics e.g. `170` will not match `170.00`.
+* Searching for guests using partial field characters e.g. `170` will **not** match `170.00`.
 
 ### Deleting a guest : `delete`
 
-This command deletes the specified guest from GuestBook. 
+This command deletes the specified guest _(with index in different scenario)_ from GuestBook. 
 
-This command is typically used when a guest has checked out of the hotel. Do note that this command 
-is irreversible. Once deleted, the specific guest's data cannot be retrieved. Please use this 
+This command is supposed to be used when a guest has checked out of the hotel. Do note that the `delete` command 
+is **IRRIVERSIBLE**. Once deleted, the specific guest's data cannot be retrieved. Please use this 
```

Comment:
I think we can keep "typically" as the user can use it for wrong entries as well. Also, typo in "IRREVERSIBLE".